docs/build.md

```diff
@@ -76,8 +76,27 @@ editing `BIT_RATE_DEFAULT` in `app/include/user_config.h`:
 Note that, by default, the firmware runs an auto-baudrate detection algorithm so that typing a few characters at boot time will cause
 the firmware to lock onto that baud rate (between 1200 and 230400).
 
-### Integer build
-By default a build will be generated supporting floating-point variables.
+### Double build (LUA 5.3 only)
+By default a build will be generated supporting floating-point variables (floats) and integers.
+To increase the precision of the floating point variables, a double build can be created. This
+is also the default in the LUA 5.1 builds. The downside is that more memory is consumed when
+storing variables.
+You can change this
+either by uncommenting `LUA_NUMBER_DOUBLE` in `app/include/user_config.h`:
+
+```c
+#define LUA_NUMBER_DOUBLE
+```
+
+OR by overriding this with the `make` command as it's [done during the CI
+build](https://github.com/nodemcu/nodemcu-firmware/blob/master/.travis.yml#L30):
+
+```
+make EXTRA_CCFLAGS="-DLUA_NUMBER_INTEGRAL ....
+```
+
+### Integer build (LUA 5.1 only)
```

Comment:
```suggestion
### Integer build (Lua 5.1 only)
```

Reply:
I fixed everything so that it was of the form "Lua 5.1" or "Lua 5.3"